Shared/Samples/Show WFS layer with XML query/ShowWFSLayerWithXMLQueryView.swift

```diff
@@ -0,0 +1,135 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowWFSLayerWithXMLQueryView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// Map with the Topographic basemap style.
+    @State private var map = Map(basemapStyle: .arcGISTopographic)
+    
+    /// Create a WFS (Web Feature Service) feature table using a specified URL and table name.
+    @State private var seattleTreesTable = WFSFeatureTable(
+        url: .wfsUrl, // The URL to the WFS service.
+        tableName: .seattleTreesDowntown // The name of the table within the service.
+    )
+    
+    /// Flag to indicate whether data is currently being loaded.
+    @State private var isLoading = false
+    
+    var body: some View {
+        MapViewReader { mapView in
+            MapView(map: map)
+            // Overlay with loading view in the center if data
+            // is currently being loaded.
+                .overlay(alignment: .center) {
+                    if isLoading {
+                        loadingView
+                    }
+                }
+                .task {
+                    do {
+                        // Load the WFS data.
+                        try await loadData()
+                        if let extent = seattleTreesTable.extent, !isLoading {
+                            // Animate the map to the extent of the data over 1 second.
+                            await mapView.setViewpoint(
+                                Viewpoint(boundingGeometry: extent),
+                                duration: 1.0
+                            )
+                        }
+                    } catch {
+                        // If an error occurs during loading, capture it to trigger an alert.
+                        self.error = error
+                    }
+                }
+            // Display an alert if there is an error during data loading.
+                .errorAlert(presentingError: $error)
+        }
+    }
+    
+    /// Asynchronous function to load data from the WFS service
```

Comment:
```suggestion
    /// Load data from the WFS service.
```
The asynchronous nature of the method is indicated by the `async` keyword, no need to repeat.